### PR TITLE
Update and align the code of service y2r!

### PR DESCRIPTION
**Pull request by JamePeng**
This change will fixed the Application EX Troopers(エクストルーパーズ) some Graphics problem！

### CODE REVIEW

**Comment by neobrain** [comment]
Hi, this is neobot, using neobrain's account. I'm keeping an archive of versions of this PR:

2016-02-26T20:10:07Z: https://github.com/citra-emu/citra/compare/2f56125f12ed2a9a18226f968b44d0e0a36bdd91...JamePeng:999f48b9d656b514cfc652fa5d8a59e085cd9957

**Comment by JamePeng** [comment]
Before(Only yellow  screen):
qq 20160227040819

After:
qq 20160227034957
qq 20160227040400
qq 20160227040411
qq 20160227035242

File: src/core/hle/service/y2r_u.cpp [hunk 1]
```diff
@@ -82,6 +82,9 @@ ResultCode ConversionConfiguration::SetStandardCoefficient(StandardCoefficient s
     return RESULT_SUCCESS;
 }
 
+/**
+ * Y2R_U::SetInputFormat service function
```
Comment (lioncash) [inline src/core/hle/service/y2r_u.cpp]: There's no need for comments like this, they provide no extra information that isn't already known

File: src/core/hle/service/y2r_u.cpp [hunk 2]
```diff
@@ -118,6 +166,21 @@ static void SetBlockAlignment(Service::Interface* self) {
     cmd_buff[1] = RESULT_SUCCESS.raw;
 }
 
+/**
+ * Y2R_U::GetBlockAlignment service function
+ */
+static void GetBlockAlignment(Service::Interface* self) {
+    u32* cmd_buff = Kernel::GetCommandBuffer();
+
+    LOG_DEBUG(Service_Y2R, "called Get alignment=%hhu", conversion.block_alignment);
```
Comment (lioncash) [inline src/core/hle/service/y2r_u.cpp]: GetBlockAlignment

File: src/core/hle/service/y2r_u.cpp (hunk 1 above)
Comment (JamePeng) [inline src/core/hle/service/y2r_u.cpp]: @lioncash 
Oh! Should I delete the comment which have no extra information

File: src/core/hle/service/y2r_u.cpp (hunk 2 above)
Comment (lioncash) [inline src/core/hle/service/y2r_u.cpp]: block_alignment=%hhu

File: src/core/hle/service/y2r_u.cpp [hunk 3]
```diff
@@ -230,6 +326,21 @@ static void SetInputLines(Service::Interface* self) {
     cmd_buff[1] = conversion.SetInputLines(cmd_buff[1]).raw;
 }
 
+/**
+ * Y2R_U::GetInputLines service function
+ */
+static void GetInputLines(Service::Interface* self) {
+    u32* cmd_buff = Kernel::GetCommandBuffer();
+
+    LOG_DEBUG(Service_Y2R, "called input_line_number=%u", conversion.input_lines);
```
Comment (lioncash) [inline src/core/hle/service/y2r_u.cpp]: input_lines=%u

**Comment by neobrain** [comment]
Hi, this is neobot, using neobrain's account. I'm keeping an archive of versions of this PR:

2016-02-26T20:36:18Z: https://github.com/citra-emu/citra/compare/2f56125f12ed2a9a18226f968b44d0e0a36bdd91...JamePeng:f488fac5763eacebfbc0cc6258492a78472b522e

**Comment by yuriks** [comment]
Can you please not do the re-alignment? I think it's unnecessary. At the very least separate it into a separate commit, but I'd do away with it entirely.

Others in the team may disagree here, but I think that over-zealous aligning of code just leads to more annoyance in the long run. It usually ends up mis-aligned again as soon as someone adds a new field.

**Comment by JamePeng** [comment]
@yuriks :p
How about separate the align operation to another commit

**Comment by purpasmart96** [comment]
IMO I actually don't mind the realignment, but if your gunna do that you should also realign the function table.

**Comment by JamePeng** [comment]
@purpasmart96 
Ok!

File: src/core/hle/service/y2r_u.cpp [hunk 4]
```diff
@@ -250,6 +375,46 @@ static void SetStandardCoefficient(Service::Interface* self) {
...
+    case StandardCoefficient::ITU_Rec601_Scaling:
+        index = static_cast<u32>(StandardCoefficient::ITU_Rec601_Scaling);
+        break;
+    case StandardCoefficient::ITU_Rec709_Scaling:
+        index = static_cast<u32>(StandardCoefficient::ITU_Rec709_Scaling);
+        break;
+    }
+
+    if (index >= 0 && index <= 3)
+    {
```
Comment (purpasmart96) [inline src/core/hle/service/y2r_u.cpp]: Our coding style states that the curly braces be like this
if (index >= 0 && index <= 3) {
not 
if (index >= 0 && index <= 3)
{

Comment (JamePeng) [inline src/core/hle/service/y2r_u.cpp]: @purpasmart96 
if (index >= 0 && index <= 3) {
} else {
}

or
if (index >= 0 && index <= 3) {
} 
else {
}

**Comment by neobrain** [comment]
Hi, this is neobot, using neobrain's account. I'm keeping an archive of versions of this PR:

2016-02-27T17:44:49Z: https://github.com/citra-emu/citra/compare/2f56125f12ed2a9a18226f968b44d0e0a36bdd91...JamePeng:0b636d1fd185c91fe40ff63432b8589755f808e7

File: src/core/hle/service/y2r_u.cpp (hunk 4 above)
Comment (purpasmart96) [inline src/core/hle/service/y2r_u.cpp]: The former

**Pull request closed without merging**